polygeist/tools/cgeist/Test/Verification/sycl/constructors.cpp

```diff
@@ -17,13 +17,14 @@
 // CHECK-DAG: !sycl_range_1_ = !sycl.range<1>
 
 // Ensure the constructors are NOT filtered out, and sycl.cast is generated for cast from sycl.id or sycl.range to sycl.array.
-// CHECK:      func.func @_ZN4sycl3_V12idILi1EEC1ERKS2_(%arg0: memref<?x!sycl_id_1_>, %arg1: memref<?x!sycl_id_1_>) attributes {llvm.linkage = #llvm.linkage<linkonce_odr>} {
+// CHECK-LABEL: func.func @_ZN4sycl3_V12idILi1EEC1ERKS2_(%arg0: memref<?x!sycl_id_1_>, %arg1: memref<?x!sycl_id_1_>)
+// CHECK-SAME:   attributes {llvm.cconv = #llvm.cconv<spir_funccc>, llvm.linkage = #llvm.linkage<linkonce_odr>, passthrough = ["norecurse", "nounwind", "convergent", "mustprogress"]} {
```

Comment:
can we name these attributes as e.g., `[[SPIRFUNC]]` and use that for all spir functions?
(same for other lits)

Reply:
done